shared/utils/src/test/roundvalue.unit.spec.js

```diff
@@ -9,51 +15,121 @@ tape('\n', function (test) {
 	test.end()
 })
 
-tape('roundValue tests', function (test) {
-	let value = 1.23456789e-10
-	let rounded = 1.2e-10
-	test.equal(roundValueAuto(value), rounded, `should return ${rounded}`)
+tape('roundValueAuto()', function (test) {
+	let value, rounded, formatted
+
+	value = 1.23456789e-10
+	rounded = 1.2e-10
+	test.equal(roundValueAuto(value), rounded, `Should return rounded value=${rounded} for input value=${value}`)
 
 	value = 1.2345
 	rounded = 1.23
-	test.equal(roundValueAuto(value), rounded, `should return ${rounded}`)
+	test.equal(roundValueAuto(value), rounded, `Should return rounded value=${rounded} for input value=${value}`)
+
+	value = 10549.23556789
+	rounded = 10549.24
+	formatted = '1.1e+4'
+	test.equal(roundValueAuto(value), rounded, `Should return rounded value=${rounded} for input value=${value}`)
+	test.equal(
+		roundValueAuto(value, true),
+		formatted,
+		`Should return formatted value=${formatted} for input value=${value}`
+	)
+
+	value = 1549.23556789
+	rounded = 1549.24
+	test.equal(roundValueAuto(value), rounded, `Should return rounded value=${rounded} for input value=${value}`)
+
+	value = 549.23556789
+	rounded = 549.24
+	test.equal(roundValueAuto(value), rounded, `Should return rounded value=${rounded} for input value=${value}`)
+
+	value = -89378.345862
+	rounded = -89378.35
+	formatted = '-8.9e+4'
+	test.equal(roundValueAuto(value), rounded, `Should return rounded value=${rounded} for input value=${value}`)
+	test.equal(
+		roundValueAuto(value, true),
+		formatted,
+		`Should return formatted value=${formatted} for input value=${value}`
+	)
+
+	value = -0.006
+	test.equal(roundValueAuto(value), value, `Should return input value=${value} unchanged`)
+
+	value = 1000
+	test.equal(roundValueAuto(value), value, `Should return input value=${value} unchanged`)
+
+	value = 1000.1234
+	rounded = 1000.12
+	test.equal(roundValueAuto(value), rounded, `Should return rounded value=${rounded} for input value=${value}`)
+
+	value = -0.0001
+	test.equal(roundValueAuto(value), value, `Should return rounded value=${rounded} for input value=${value}`)
```

Comment:
test msg typo, should be value but not `rounded`

Reply:
Fixed in https://github.com/stjude/proteinpaint/pull/2439/commits/9bb2c0ec26cef29c6a8c36b08018b55bf3246881